training/tf/chunkparser.py

```diff
@@ -31,6 +31,7 @@
 import threading
 import time
 import unittest
+from config import *
 
 # 16 planes, 1 side to move, 1 x 362 probs, 1 winner = 19 lines
```

Comment:
leelaz's source also allows arbitrary odd board size, but there remain comments in CPUPipe.cpp and NNCache.h that have `19` in them. I think these are not wrong since the board size is defaulted to 19 in config.h; also concrete numbers are helpful. We could point out somewhere that these comments are using the 19x19 board as an example.

Reply:
Fair point. I'll keep that in mind when I get around to it.